### ISSUE TITLE
Problems importing nltk3 after PunktWordTokenizer removal

### ISSUE THREAD

**Issue by hcorreasuarez** [nltk/nltk#752]
``` python
import nltk
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/nltk-3.0.0-py2.7.egg/nltk/__init__.py", line 117, in <module>
    from nltk.align import *
  File "/usr/local/lib/python2.7/dist-packages/nltk-3.0.0-py2.7.egg/nltk/align/__init__.py", line 15, in <module>
    from nltk.align.ibm1 import IBMModel1
  File "/usr/local/lib/python2.7/dist-packages/nltk-3.0.0-py2.7.egg/nltk/align/ibm1.py", line 18, in <module>
    from nltk.corpus import comtrans
  File "/usr/local/lib/python2.7/dist-packages/nltk-3.0.0-py2.7.egg/nltk/corpus/__init__.py", line 64, in <module>
    from nltk.tokenize import RegexpTokenizer
  File "/usr/local/lib/python2.7/dist-packages/nltk-3.0.0-py2.7.egg/nltk/tokenize/__init__.py", line 69, in <module>
    from nltk.tokenize.punkt    import PunktSentenceTokenizer, PunktWordTokenizer
ImportError: cannot import name PunktWordTokenizer
```

**Comment by alvations** [comment]
@hcorreasuarez are you still having the same error after upgrading to the latest version of NLTK? Try `pip install -U nltk` to upgrade your NLTK version.

**Comment by alvations** [comment]
Shouldn't be a proper on the latest version. @hcorreasuarez please reopen if problem still persist.

Closing resolved issues to prepare for v3.2.5.
